lib/Form/styles.scss

```diff
@@ -94,16 +94,24 @@ $form-checkbox-label-size: $typo-size-lead !default;
 .form-radiobutton--invisible {
   visibility: hidden;
 }
-
+form .dropdown-gc .button--outline,
```

Comment:
Should this be `.form`? Also, why do we need to target the dropdown module?

Reply:
our form component does not have that class...

& this is why: 
![Screen Shot 2019-07-10 at 16 30 53](https://user-images.githubusercontent.com/12775966/60982678-819fc500-a330-11e9-870a-29e51f458608.png) now the dropdowns look like the other inputs